### PR TITLE
Added Change Background

### LINKED ISSUE #14: Add 'Change Background' Item in Desktop Context Menu
`FilesContainer.vue` has a default context menu list, but we can get list of custom menu items from props and put them alongside of default ones.

### PR DESCRIPTION
**Pull request by asheghi**
an attempt to add Change Background (fixes #14 )

what i've done:
- added ChangeBackground.exe to programs
- added ChangeBackground to Context Menu of Desktop
- added ChangeBackground shortcut to Desktop
- implemented basic functionality to change and remember wallpaper

### CODE REVIEW

**Comment by nainemom** [comment]
Nice job @semycolon!

File: src/styles/constants.js [hunk 1]
```diff
@@ -2,3 +2,20 @@ export const panelSize = '46px';
 
 export const contextMenuWidth = '180px';
 export const contextMenuItemHeight = '30px';
+
+
+export const wallpapers = [
+  import('../assets/wallpapers/1.jpg'),
```
Comment (nainemom) [inline src/styles/constants.js]: Would you please move the wallpaper files to the `/files` directory and handle upcomming stuff? By doing that, wallpaper files also accessible via `$fs` and `My Computer.exe` program.

File: src/components/FilesContainer.vue [hunk 2]
```diff
@@ -141,6 +145,7 @@ export default {
           'Refresh',
           ...(this.staticPath ? ['Create New Folder', 'Create New Text File'] : []),
           ...(this.staticPath && (this.$wm.markedFiles.copyList.length || this.$wm.markedFiles.cutList.length) ? ['Paste'] : []),
+          ...(this.isInDesktop ? ['Change Background'] : []),
```
Comment (nainemom) [inline src/components/FilesContainer.vue]: Can you please handle custom context menu items like this in `Desktop.vue` component? Desktop component using this component inside, so You can pass it via custom property.

File: src/components/Os.vue [hunk 3]
```diff
@@ -57,6 +57,7 @@ export default {
     },
   },
   errorCaptured(e) {
+    console.error(e);
```
Comment (nainemom) [inline src/components/Os.vue]: i just updatet the `master` branch to fix `lint` and `lint:fix` scripts. please rebase and check the lint problems.

**Comment by asheghi** [comment]
@nainemom thanks, but would you please do a code review, if it is enough, merge it, otherwise let me know what changes are required

**Review by nainemom — COMMENTED** [review]
> @nainemom thanks, but would you please do a code review, if it is enough, merge it, otherwise let me know what changes are required

I already opened some discussions on your changes.

File: src/components/FilesContainer.vue (hunk 2 above)
Comment (asheghi) [reply src/components/FilesContainer.vue]: sure checkout this commit 8cf22f9c2a810b1a9092403c6342a6523ee90583

File: src/components/Os.vue (hunk 3 above)
Comment (asheghi) [reply src/components/Os.vue]: resolved by 7a080c8.
I think it's necessary to use console.error to get more information about the error.

File: src/styles/constants.js (hunk 1 above)
Comment (asheghi) [reply src/styles/constants.js]: resolved by 95e4d14 , but it raised the files import size to 20MB. How should we proceed with the file import size issue? Should we resolve it in this thread or do it later?

Comment (nainemom) [reply src/styles/constants.js]: As we discussed earlier, that can be handled by "network-fs" or something like that.

**Comment by nainemom** [comment]
@semycolon I've made some changes on your pr and it's almost ready to merge. please pull  `https://github.com/nainemom/win7/tree/semycolon-adding_change_background` on this pr branch.

**Pull request merged**